rt/test/bash/testcmd3

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#######################################################################
+# z390 - Mainframe assembler emulator and run-time engine
+# Copyright (C) 2021 z390 Assembler LLC
+#
+# This file is part of z390.
+# z390 is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# z390 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <https://www.gnu.org/licenses/>.
+#######################################################################
+echo "echo testcmd3: bash command executed by TESTCMD3.MLC via SVC 34"
+echo "echo Hello from testcmd3"
+echo "echo Hello again"
+echo "echo Hello for the last time"
+echo "REM /* END testcmd3 bash command executed by TESTCMD3.MLC"
```

Comment:
These echo scripts (bash/testcmd3, bash/testcmd4[abc]) all seem the same except for the different program name in the text. Could we consolidate these to a single script with a parameter that changes the echo text rather than having separate scripts? Same thing for the BAT scripts.

Reply:
TESTCMD3 and TESTCMD4 test the use of the CMDPROC macro. CMDPROC starts a Command Processor (CP); for Windows, this is cmd.exe; for Linux/Mac, this is sh. TESTCMD3 starts one CP, feeds it the command rt/test/bat/TESTCMD3.BAT or rt/test/bash/testcmd3, then reads the responses of that command, writing the responses to the console. TESTCMD4 starts two CPs, feeds each of them a command, then reads the responses and writes the responses to the console. Yes, all the commands tested just write comments (REM statements for Windows, echo statements for Linux/Mac). The tests were modeled on the original tests TESTCMD1 and TESTCMD2. The CP created in TESTCMD1 runs TESTCMD1.BAT and the two CPs created in TESTCMD2 run TESTCMDA.BAT and TESTCMDB.BAT. If you look at these BAT files and the new bat/bash files, you will see that the new tests are modeled after these BAT files.

Yes, we could probably consolidate them, but I don't think it's worth it for these simple tests.

Reply:
Ok. Not a show stopper.